conjure-lib/src/main/java/com/palantir/conjure/java/lib/Bytes.java

```diff
@@ -113,6 +115,21 @@ public static Bytes from(ByteBuffer buffer) {
         return new Bytes(safe);
     }
 
+    /** Constructs a new {@link Bytes} from the provided {@link ByteArrayOutputStream} */
+    public static Bytes from(ByteArrayOutputStream byteStream) {
```

Comment:
in our internal product we have quite a few places that would use this, and I think this is a lot more common than ByteBuffer

Reply:
I can extend `ByteArrayOutputStream` overriding `toByteArray` in a way that breaks the immutability of this class.

Reply:
hmm. I didn't think about that. I suppose if you want to guarantee safety you could make an additional copy if the runtime class is a subclass of ByteArrayOutputStream. this might be too complicated now though